File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/search_history_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class SearchHistoryVO:
    """搜索历史响应"""
    created_at: str = None
    updated_at: str = None
    keyword: str = None
    search_time: str = None
    count: int = None

File: sdkwork-app-sdk-typescript/src/types/search-history-vo.ts
/** 搜索历史响应 */
export interface SearchHistoryVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 关键词 */
  keyword?: string;
  /** 搜索时间 */
  searchTime?: string;
  /** 搜索次数 */
  count?: number;
}
